pkg/compute/serve.go

```diff
@@ -0,0 +1,313 @@
+package compute
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/blang/semver"
+	"github.com/fastly/cli/pkg/cmd"
+	"github.com/fastly/cli/pkg/compute/manifest"
+	"github.com/fastly/cli/pkg/config"
+	"github.com/fastly/cli/pkg/errors"
+	fstexec "github.com/fastly/cli/pkg/exec"
+	"github.com/fastly/cli/pkg/filesystem"
+	"github.com/fastly/cli/pkg/text"
+	"github.com/fastly/cli/pkg/update"
+)
+
+// ServeCommand produces and runs an artifact from files on the local disk.
+type ServeCommand struct {
+	cmd.Base
+
+	build            *BuildCommand
+	env              cmd.OptionalString
+	file             string
+	force            cmd.OptionalBool
+	includeSrc       cmd.OptionalBool
+	lang             cmd.OptionalString
+	manifest         manifest.Data
+	name             cmd.OptionalString
+	skipBuild        bool
+	viceroyVersioner update.Versioner
+}
+
+// NewServeCommand returns a usable command registered under the parent.
+func NewServeCommand(parent cmd.Registerer, globals *config.Data, build *BuildCommand, viceroyVersioner update.Versioner) *ServeCommand {
+	var c ServeCommand
+
+	c.build = build
+	c.viceroyVersioner = viceroyVersioner
+
+	c.Globals = globals
+	c.CmdClause = parent.Command("serve", "Build and run a Compute@Edge package locally")
+
+	c.manifest.File.SetOutput(c.Globals.Output)
+	c.manifest.File.Read(manifest.Filename)
+
+	c.CmdClause.Flag("env", "The environment configuration to use (e.g. stage)").Action(c.env.Set).StringVar(&c.env.Value)
+	c.CmdClause.Flag("file", "The Wasm file to run").StringVar(&c.file)
```

Comment:
Am interested in knowing why you didn't set the default via the flag itself. I personally prefer this method, as the default value then gets documented rather than being magic. 

```suggestion
	c.CmdClause.Flag("file", "The Wasm file to run").Default("bin/main.wasm").StringVar(&c.file)
```

Reply:
I didn't know that was a thing with Kingpin. So I'll sort that 👍🏻